src/traces/bar/attributes.js

```diff
@@ -35,8 +42,32 @@ module.exports = {
     y: scatterAttrs.y,
     y0: scatterAttrs.y0,
     dy: scatterAttrs.dy,
+
     text: scatterAttrs.text,
 
+    textposition: {
```

Comment:
What would be the meaning of `'total'` for inner bars? no label? the bar size? the cum sum up to that bar?